sysdig/internal/client/v2/client.go

```diff
@@ -65,8 +65,9 @@ type Requester interface {
 }
 
 type Client struct {
-	config    *config
-	requester Requester
+	config          *config
+	requester       Requester
+	policiesChanged bool
```

Comment:
Again we have specific code related to policies in Client which is completely abstract.

Reply:
Modified to move policies specific code out of the abstract objects.